apps/devportal/data/markdown/pages/learn/faq/xm-personalize/features.md

```diff
@@ -6,45 +6,35 @@ hasSubPageNav: true
 cdpTags: ['xm', 'personalize']
 ---
 
-**Once I build a XM + Personalize site and deploy it, will I be able to port this site to XM Cloud when that comes out? Do you have a playbook available that we could see?**
+### Once I build a XM + Personalize site and deploy it, will I be able to port this site to XM Cloud when that comes out? Do you have a playbook available that we could see?

```

Comment:
This sentence mentions `to XM Cloud when that comes out?` Probably should just rephrase.

Reply:
My next commit will include a removal of that part. The sentence works without it